### ISSUE TITLE
Hatem Enaami

### ISSUE THREAD

**Issue by aflies1** [promazo/Sourcing#96]
RIT - Phd Engineering and Comp Sci

**Comment by aflies1** [comment]
Extensive background in managing IT projects over the last 15 years
Has some technical skills, but I don't think at a high enough level that we may need, will need Louis/Alberto to confirm
Does have an accent but no issues with speaking/understanding English (no time did I need to have him repeat anything)
Has finished all of his course work, worked with RIT, worked with the University for approval to work a full-time position moving forward. 
Is OK only managing the Schwab work, more than willing to take on more as well
Very familiar with SCRUM/AGILE
Managed similar projects to the schwab work with the central bank of Libya

**Comment by aflies1** [comment]
https://docs.google.com/document/d/1wgNd5SDyUIyapW7GoZ-4rdUnSFyf1I71/edit#heading=h.gjdgxs

**Comment by jaahmuhl** [comment]
Strong. Good natured. Seems to have a lot of experience with both students and project management.

Knows scrum incredibly well. Is looking forward to helping students.

Only issue is he can sometimes go on a tangent and not always answer the question precisely. Seems to not every time understand the question being asked.

**Comment by ldaudet** [comment]
Hatem

He's at Rochester IT, doing an electrical and communication PhD working on point cloud compression technique. It's very involved in C++ code.

He got his bachelor degree from Libya, where he was one of the top 5% engineer in electrical and electronic engineering. He then worked the field for a while, working in telecom, and server setup. He also created a company called PMO. He was also the assistent project manager on Wi-Max (a big tech from Alcatel). He then did his master in electrical and electronic engineering in 2012 with an IT concentration (4.0 GPA).

He designed RFPs (a technical SOW of sorts) both for the central bank of lybia while they overhauled the system, moving from IBM to oracle cloud.

His technical background is old by his own admission. He's from the generation of assembler and Fortran, and then was able to move on to C/C++.

In term of pm framework, he used to use waterfall for the engineering projects, but then when he moved to the banking sector, he moved to SCRUM and got aquainted to it.

He prefer to work in smaller teams. When having huge team, he prefers to break it up into smaller team, and then work carefully to select a team lead to each of these teams so that he can delegate.

He's done with his PhD courswork. His GPA being 4.0, he was allowed by his advisor to look for a new work on the side. He wants it to keep his skills sharp.

He is interested in getting his programing skills developed further. He has a very little bit of schooling on Machine learning. 

During the course of his work experience, he took a few classes on project management. He worked on multiple projects at once, he worked on turn key projects, and projects of both engineering and IT concentration. He is also fine with traveling, writting up SOWs and sales.

In terms of dev, he doesn't have a strong skillset. He's got C++ from an engineer perspective, and that's about it. No Python, java, ML, web dev, etc...

TL;DR: For a pm perspective, I still see him as somewhat of an odd ball, in part from his english (his english is good, but full of strange expressions), in part because like Jason on USAA he can chat a lot, and in part because he often answer questions slightly off (like asking him what he would like to learn in IT if he joined promazo, as we have a lot of different projects, and his answer was that he would love to learn dev languages, and that he was very confortable with project management, rather than talking about IT application like ML or becoming full stack). However, besides his somewhat oddness, he has a clearly large resume under his belt (albeit most of his experience were international, which has a pretty strong influence on PMing IMO). His experience managing part of a fairly large project at rochester is to me his most valuable experience from a pm perspective, showing that he can manage a project with a team states side, as well as with a team of students matching our fellows demographics.
